File: src/main/java/net/hectus/neobb/turn/default_game/structure/glass_wall/TOrangeGlassWall.java
package net.hectus.neobb.turn.default_game.structure.glass_wall;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.structure.PlacedStructure;
import net.hectus.neobb.turn.default_game.attributes.clazz.HotClazz;

public class TOrangeGlassWall extends GlassWallTurn implements HotClazz {
    public TOrangeGlassWall(NeoPlayer player) { super(player); }
    public TOrangeGlassWall(PlacedStructure data, NeoPlayer player) { super(data, player); }

    @Override
    public String color() {
        return "orange";
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/default_game/structure/glass_wall/TOrangeGlassWall.kt
package net.hectus.neobb.modes.turn.default_game.structure.glass_wall

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.matrix.structure.PlacedStructure
import net.hectus.neobb.matrix.structure.StaticStructure
import net.hectus.neobb.matrix.structure.StaticStructures
import net.hectus.neobb.modes.turn.default_game.attribute.clazz.HotClazz
import net.hectus.neobb.player.NeoPlayer

class TOrangeGlassWall(data: PlacedStructure?, cord: Cord?, player: NeoPlayer?) : GlassWallTurn(data, cord, player), HotClazz {
    override val staticStructure: StaticStructure = StaticStructures.Default.GlassWall.ORANGE
}
